src/Stripe/Controllers/PaymentIntentRequest.php

```diff
@@ -124,10 +128,68 @@ private function getValidatedData(): PaymentIntentForm
         $data['lastName'] = !empty($postData['lastName']) ? sanitize_text_field($postData['lastName']) : null;
         $data['currency'] = !empty($postData['currency']) ? sanitize_text_field($postData['currency']) : null;
         $data['liveMode'] = !empty($postData['liveMode']) ? $postData['liveMode'] : null;
+        $data['enableLink'] = !empty($postData['enableLink']) ? $postData['enableLink'] : false;
+        $data['recaptchaToken'] = !empty($postData['recaptchaToken']) ? $postData['recaptchaToken'] : null;
 
         return PaymentIntentForm::fromArray($data);
     }
 
+    /**
+     * @param $data
+     * @return bool|void
+     */
+    public function validateRecaptcha($data)
+    {
+
+        // Verify the captcha has been there's a token.
+        if (!$data->recaptchaToken) {
```

Comment:
I suggest using `if (empty($data->recaptchaToken))` as it will also catch of the key isn't set rather than throwing an error.

Reply:
1. Yes, that's the way it works.
2. Correct

I've changed the conditional to use `empty()` here: https://github.com/impress-org/donation-block-for-stripe/pull/53/commits/d3268f90454274c1ac579dd9aed9591e51eb4331